CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (tx) [#12659](https://github.com/cosmos/cosmos-sdk/pull/12659) Remove broadcast mode `block`.
```

Comment:
This is a big change IMO, it would maybe be useful to put it in Client-breaking and CLI-breaking too

Reply:
will add it. Also a upgrading.md entry is needed